aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1852,6 +1811,89 @@ static Tensor& linalg_norm_out_impl(Tensor& result, const Tensor& self, optional
   return result;
 }
 
+static Tensor& linalg_vector_norm_impl(Tensor& result, const Tensor& self, optional<Scalar> opt_ord, optional<IntArrayRef> opt_dim, bool keepdim, optional<ScalarType> opt_dtype) {
+  auto ord = opt_ord.value_or(2).toDouble();
```

Comment:
What happens if ord is a large integer value not exactly representable as a double?

Reply:
Per offline discussion, let's just add a comment